src/main/java/stream/flarebot/flarebot/commands/FlareBotManager.java

```diff
@@ -252,32 +247,26 @@ public void saveLocalisation() {
                     PreparedStatement statement = conn
                             .prepareStatement("INSERT INTO localisation (guild_id, locale) VALUES (?, ?) ON DUPLICATE KEY locale = VALUES(locale)");
                     statement.setString(1, s);
-                    statement.setString(2, locale.get(s).toLanguageTag());
+                    statement.setString(2, locale.get(s).toString());
                     statement.execute();
                 });
             } catch (SQLException e) {
                 e.printStackTrace();
             }
         }
-        for (Config localeFile : localeFiles.values()) {
-            localeFile.save();
-        }
     }
 
-    public String getLang(Language lang, String id) {
-        if (locale.containsKey(id)) {
-            Locale l = locale.get(id);
-            String path = lang.name().toLowerCase().replaceAll("\\_", ".");
-            Config config = localeFiles.get(l);
-            return config.getString(path);
-        } else {
-            locale.put(id, Locale.UK);
-            String path = lang.name().toLowerCase().replaceAll("\\_", ".");
-            Config config = localeFiles.get(Locale.UK);
-            return config.getString(path);
-        }
+    public LocalConfig loadConfig(Language.Locales l) {
+        ClassLoader cl = FlareBot.class.getClassLoader();
```

Comment:
Instead of the class loader use the class itself and for the love of code stop doing one use variables.